src/commands/leaderboard.py

```
@@ -23,107 +24,406 @@
+        cutoff = datetime.now(timezone.utc) - timedelta(days=days)
+        if hasattr(Pick, "timestamp"):
+            query = query.where(getattr(Pick, "timestamp") >= cutoff)
+    if contest_id is not None and hasattr(Pick, "contest_id"):
+        query = query.where(getattr(Pick, "contest_id") == contest_id)
+
+    # Only include picks that have been resolved as correct when a
+    # status field is available. This mirrors the previous behaviour
+    # that filtered on Pick.status == "correct", rather than relying
+    # solely on the is_correct flag.
+    if hasattr(Pick, "status"):
+        query = query.where(getattr(Pick, "status") == "correct")
```

Ruff (0.14.10)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)